Source_Code/Server/ClueLessServer.java

```diff
@@ -287,7 +287,13 @@ public void processAction(ActionRequest actionRequest)
                 {
                     processGameStartRequest((GameStartRequest) actionRequest);
                 }
-                if(PlayerList.get( CurrentPlayerIndex ).PlayerName.equals(p.PlayerName))  // Only the player who's turn it is can do these
+                
+                else if(actionRequest instanceof RefuteSuggestionResponse)
```

Comment:
Since this is coming from a player who's turn it is NOT, this needs to be separated from the main if loop below.